stock_reserve_rule/models/stock_reserve_rule.py

```diff
@@ -152,6 +154,96 @@ class StockReserveRuleRemoval(models.Model):
         "When empty, any packaging can be removed.",
     )
 
+    TOLERANCE_LIMIT = [
+        ("no_tolerance", "No tolerance"),
+        ("upper_limit", "Upper Limit"),
+        ("lower_limit", "Lower Limit"),
+    ]
+
+    tolerance_requested_limit = fields.Selection(
+        selection=TOLERANCE_LIMIT,
+        string="Tolerance on",
+        help="Selecting a tolerance limit type"
+        "No tolerance: lot qty should be equal required"
+        "Upper Limit: lot with higher qty than required"
```

Comment:
Make help independent of lot
```suggestion
        "Upper Limit: quantity higher than demand but within tolerance"
```

Reply:
Resolved